src/components/FindUs/createMapOptions.ts

```diff
@@ -1,4 +1,8 @@
-export const createMapOptions = () => ({
+interface createMapArguments {
```

Comment:
[Style] It's generally convention to PascalCase type names, not camelCase.

```suggestion
interface CreateMapArguments {
```